homepage/homepage/content/docs/using-covalues/subscription-and-loading.mdx

```diff
@@ -465,6 +465,190 @@ project?.owner // => always null
 The load operation will succeed and return the object, but the inaccessible reference will always be `null`.
 
 
+#### Deep loading lists with shared items
+
+When loading a list with shared items, you can use the `$onError` option to safely load the list skipping any inaccessible items.
+
+This is especially useful when in your app access to these items might be revoked.
+
+This way the inaccessible items are replaced with `null` in the returned list.
+
+<CodeGroup>
+```typescript twoslash
+import { ID, CoMap, CoList, co, Group, Account } from "jazz-tools";
+import { expect, assert } from "vitest";
+
+class Person extends CoMap {
+  name = co.string;
+}
+class Friends extends CoList.Of(co.ref(Person)) {}
+
+const privateGroup = Group.create();
+const publicGroup = Group.create();
+const me = {} as unknown as Account;
+
+// ---cut-before---
+const source = Friends.create(
+  [
+    Person.create(
+      {
+        name: "Jane",
+      },
+      privateGroup, // We don't have access to Jane
+    ),
+    Person.create(
+      {
+        name: "Alice",
+      },
+      publicGroup, // We have access to Alice
+    ),
+  ],
+  publicGroup,
+);
+
+const friends = await Friends.load(source.id, {
+  resolve: {
+    $each: { $onError: null }
+  },
+  loadAs: me,
+});
+
+// Thanks to $onError catching the errors, the list is loaded
+// because we have access to friends
+expect(friends).not.toBeNull();
```

Comment:
If you put the output in comments it will make it easier for the reader to connect the assertions to the result (rather than having to infer it from the surrounding comments and code)

Reply:
Done: https://github.com/garden-co/jazz/pull/2247/commits/16d5553ccd621a9f3876e1be12f56c006e19cc6f